documentation/sphinx/source/release-notes.rst

```diff
@@ -14,6 +14,8 @@ Performance
 Fixes
 -----
 
+* If a cluster is upgraded during an `onError` call, the cluster could return a `cluster_version_changed` error. `(PR #1734) <https://github.com/apple/foundationdb/pull/1734>`_.
```

Comment:
This appears to break the documentation build

> Warning, treated as error:
/__this_is_some_very_long_name_dir_needed_to_fix_a_bug_with_debug_rpms__/foundationdb/documentation/sphinx/source/release-notes.rst:17: ERROR: Broken role invoked